projects/ngx-d3timeline/src/lib/content/content.component.spec.ts

```diff
@@ -39,12 +39,14 @@ describe('ContentComponent', () => {
   it('should render correctly', () => {
     const rectangles: EventRectangle[] = [
       {
+        id: 1,
```

Comment:
doesn't this affect the snapshots?

Reply:
No the id doesn't get rendered in the html